esp-println/src/lib.rs

```diff
@@ -78,7 +78,7 @@ impl core::fmt::Write for Printer {
 }
 
 impl Printer {
-    fn write_bytes(bytes: &[u8]) {
+    pub fn write_bytes(bytes: &[u8]) {
```

Comment:
We should have API docs for public items

Reply:
Done